documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -263,3 +263,49 @@ app.listen(3000, () => {
 	console.log('listening on port 3000');
 });
 ```
+
+## Aborted requests
+
+> [!NOTE] This feature is available in `vite dev` and `vite preview` servers.
+
```

Comment:
I don't think we necessarily need to mention this here as basically all features are available during `dev` and `preview`
```suggestion
```